.circleci/README.md

```diff
@@ -2,7 +2,9 @@
 
 ## Variables
 
-See the list of possible environment variables in the follwoing table. They can be set in the project settings, or part of an organization-wide context. Without setting the **required** variables, the pipeline will fail.
+See the list of possible environment variables in the following table. They can be set in the project settings, or part of an organization-wide context. Without setting the **required** variables, the pipeline will fail.
+
+Note that the required variables are also set in the [minimal_env.sh](../bin/minimal_env.sh) script.
```

Comment:
I would not add this line to the CircleCI documentation specifically, as CircleCI doesn't source this file. `./bin/minimal_env.sh` is only relevant for use of these tools interactively on the command line. Setting variables in CircleCI is done via the UI (and maybe the API, but haven't looked into that) in the **Project Settings** --> **Environment Variables** menu.